apps/puppeteer/README.md

```
@@ -1,96 +1,317 @@
+
+## Architecture Support
+
+This service supports both AMD64 and ARM64 architectures.
+
+### Apple Silicon (M1/M2/M3) Compatibility
+
+For Apple Silicon Macs, you may encounter issues with the default Docker setup. If you see errors like:
 
-### Apple Silicon run amd64 version
-using docker desktop it only supports 32-bit emulation and resolves in error:
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)